src/examples/subscriber/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@
 px4_add_module(
 	MODULE examples__subscriber
 	MAIN subscriber
-	STACK 2400
+	STACK_MAIN 2400
```

Comment:
I thought the name was too confusing, but agreed with you, so It just issues a deprecation warning and treats stack as stack main. Already ran into this with ecl.